src/native/libs/System.Globalization.Native/pal_locale_internal.h

```diff
@@ -4,6 +4,7 @@
 #pragma once
 
 #include "pal_icushim_internal.h"
+#include "pal_localeStringData.h"
```

Comment:
do we need `pal_locale.h` here too? `pal_localeStringData.h` only has `NativeGetLocaleInfoString`.

Reply:
It is included as LocaleStringData is defined in pal_localeStringData.h and GlobalizationNative_GetLocaleInfoStringNative has a parameter type LocaleStringData.